classes/admin/class-submenu.php

```diff
@@ -460,11 +475,10 @@ protected function filterTransactions($items)
      */
     protected function getHeartlandConfiguration()
     {
-        $config = new HpsServicesConfig();
+        $config = new ServicesConfig();
         $config->secretApiKey = $this->getSetting('secret_api_key');
-        $config->versionNumber = '1510';
-        $config->developerId = '002914';
-        return $config;
+        $config->serviceUrl = "https://cert.api2.heartlandportico.com";  
```

Comment:
We need to account for both sandbox and production environments. As is, this will only send requests to sandbox, even if production credentials are configured.

Reply:
updated